.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-fc48cfe24eb99b883585ca16590a3ef32652300e
```

Comment:
I need the pin update because this change requires https://github.com/pytorch/xla/pull/5600